samples/android/15-puzzle/build.gradle.in

```diff
@@ -27,5 +28,9 @@ android {
 
 dependencies {
     //implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation project(':opencv')
+    if (gradle.opencv_source == "sdk_path") {
+        implementation project(':opencv')
+    } else if (gradle.opencv_source == "maven_local" || gradle.opencv_source == "maven_cenral") {
+        implementation 'org.opencv:opencv:@OPENCV_VERSION@'
```

Comment:
I propose to drop the version here and in all other samples.

Reply:
The version is required here